src/style.scss

```diff
@@ -233,13 +233,12 @@ slot {
       cursor: not-allowed;
       transform: unset;
 
-      color: rgba(var(-auro-color-base-black), $disabledOPacity);
+      color: rgba($auro-color-text-link-on-light, $disabledOPacity);
```

Comment:
I'd suggest using CSS versus Sass whenever we can. For this, please see https://auro.alaskaair.com/color/digital-palette#:~:text=5)%3B%0A%7D-,Using%20CSS,-Using%20CSS%20to